x-pack/plugin/core/src/main/resources/logs-default-pipeline.json

```diff
@@ -0,0 +1,24 @@
+{
+  "processors": [
+    {
+      "set": {
+        "description": "If '@timestamp' is missing, set it with the ingest timestamp",
+        "field": "@timestamp",
+        "override": false,
+        "copy_from": "_ingest.timestamp"
+      }
+    },
+    {
+      "pipeline" : {
+        "name": "logs@custom",
+        "ignore_missing_pipeline": true,
+        "description": "A custom pipeline for logs data streams, which does not exist by default"
+      }
```

Comment:
@joshdover @kpollich FYI: the `logs-*-*` index template will contain this default logs pipeline that calls out a missing `logs@custom` extension point.